sql/item_mods.sql

```diff
@@ -26606,46 +26677,86 @@ INSERT INTO `item_mods` VALUES (21707,2,78);      -- barbarity_+1 HP+78
+INSERT INTO `item_mods` VALUES (21717,25,50);     -- Pangu: Accuracy+50
+INSERT INTO `item_mods` VALUES (21717,30,50);     -- Pangu: Magic Accuracy+50
+INSERT INTO `item_mods` VALUES (21717,311,217);   -- Pangu: Magic Damage+217
+INSERT INTO `item_mods` VALUES (21719,25,20);     -- Ajja Axe: Accuracy+20
+INSERT INTO `item_mods` VALUES (21719,30,20);     -- Ajja Axe: Magic Accuracy+20
+INSERT INTO `item_mods` VALUES (21720,23,20);     -- Eletta Axe: Attack+20
+INSERT INTO `item_mods` VALUES (21720,25,30);     -- Eletta Axe: Accuracy+30
+INSERT INTO `item_mods` VALUES (21720,28,10);     -- Eletta Axe: "Magic Atk. Bonus"+10
+INSERT INTO `item_mods` VALUES (21720,30,30);     -- Eletta Axe: Magic Accuracy+30
+INSERT INTO `item_mods` VALUES (21720,311,217);   -- Eletta Axe: Magic Damage+217
+INSERT INTO `item_mods` VALUES (21721,9,10);      -- Kaja Axe: DEX+10
+INSERT INTO `item_mods` VALUES (21721,12,10);     -- Kaja Axe: INT+10
+INSERT INTO `item_mods` VALUES (21721,13,10);     -- Kaja Axe: MND+10
+INSERT INTO `item_mods` VALUES (21721,23,25);     -- Kaja Axe: Attack+25
+INSERT INTO `item_mods` VALUES (21721,25,35);     -- Kaja Axe: Accuracy+35
+INSERT INTO `item_mods` VALUES (21721,28,13);     -- Kaja Axe: "Magic Atk. Bonus"+13
+INSERT INTO `item_mods` VALUES (21721,30,35);     -- Kaja Axe: Magic Accuracy+35
+INSERT INTO `item_mods` VALUES (21721,311,217);   -- Kaja Axe: Magic Damage +217
+INSERT INTO `item_mods` VALUES (21721,642,120);   -- Kaja Axe: "Decimation" damage +120% (The "Decimation" damage boost applies to all hits in the Weapon Skill.)
+-- INSERT INTO `item_mods` VALUES (21721,??,72);  -- Kaja Axe: Main hand: "Decimation" (Grants access to Decimation regardless of skill level or completion of Axe the Competition; while equipped in the main hand only.)(This one may need to be a latent, but no latent exists for equipped in main hand)
+INSERT INTO `item_mods` VALUES (21722,9,15);      -- Dolichenus: DEX+15
+INSERT INTO `item_mods` VALUES (21722,12,15);     -- Dolichenus: INT+15
+INSERT INTO `item_mods` VALUES (21722,13,15);     -- Dolichenus: MND+15
+INSERT INTO `item_mods` VALUES (21722,23,30);     -- Dolichenus: Attack+30
+INSERT INTO `item_mods` VALUES (21722,25,40);     -- Dolichenus: Accuracy+40
+INSERT INTO `item_mods` VALUES (21722,28,16);     -- Dolichenus: "Magic Atk. Bonus"+16
+INSERT INTO `item_mods` VALUES (21722,30,40);     -- Dolichenus: Magic Accuracy+40
+INSERT INTO `item_mods` VALUES (21722,311,217);   -- Dolichenus: Magic Damage+217
+INSERT INTO `item_mods` VALUES (21722,642,120);   -- Dolichenus: "Decimation" damage +120% (The "Decimation" damage boost applies to all hits in the Weapon Skill.)
+-- INSERT INTO `item_mods` VALUES (21722,??,72);  -- Dolichenus: Main hand: "Decimation" (Grants access to Decimation regardless of skill level or completion of Axe the Competition; while equipped in the main hand only.)(This one may need to be a latent, but no latent exists for equipped in main hand)
```

Comment:
ok this is getting out of hand. once it makes me left/right scroll on a 1900 wide screen, the comment is too dang long.

Reply:
Okay, I can shorten it tomorrow. Just trying to make sure it has information needed for someone to be able to implement.